app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -183,16 +191,22 @@ public void onShareTrip(int index) {
     @Override
     public void onPlaceSelected(Place destination) {
         openTrip.setDestinationName((String) destination.getAddress());
+        saveDestination();
     }
 
+    @Override
+    public void onDestinationnHistorySelected(int id) {
+        openTrip.setDestinationName((String) userDestinationList.get(id).getDestination().getDestinationName());
+        nextPage();
+    }
 
     @Override
     public void onDateChanged(int year, int month, int day) {
         openTrip.setTripDepartureDate(year, month, day);
     }
 
     //saves landmarks into trips, called in onclick
-    public void onSave(List<LandmarkCard> saveList) {
+    public void onSave(List <LandmarkCard> saveList) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)